Cargo.toml

```diff
@@ -9,7 +9,7 @@ edition = "2018"
 [dependencies]
 tokio = "0.1.14"
 futures = "0.1.25"
-tokio-timer = "0.2.8"
+futures-timer = "0.1"
```

Comment:
Any higher version requires changing from `futures::Future` to `std::future::Future`.